modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc

```diff
@@ -23,17 +11,15 @@ data:
 
 . From the {ocp-short} web console, select the *ConfigMaps* tab.
 . Click *Create ConfigMap*.
-. From *Create ConfigMap* page, select the *YAML view* option in *Configure via* and make the changes to the file, if needed.
+. From *Create ConfigMap* page, select the *YAML view* option in *Configure via* and make changes to the file, if needed.
 . Click *Create*.
-. Go to the *Helm* tab.
-+
-The list of Helm Releases appears on the page.
-. Click the overflow menu on a Helm release and select *Upgrade*.
-. Use one of the following views to edit the Helm configuration:
+. Go to the *Helm* tab to see the list of Helm releases.
+. Click the overflow menu on the Helm release that you want to use and select *Upgrade*.
+. Use either the *Form view* or *YAML view* to edit the Helm configuration.
 
-** Using *Form view* 
+** Using *Form view*
```

Comment:
```suggestion
** Using *Form view*:
```

Reply:
Going to leave the colon off for now, since these are headings and not lead-in sentences